Rubberduck.Core/UI/Command/ComCommands/ComCommandBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using NLog;
+using Rubberduck.VBEditor.Events;
+
+namespace Rubberduck.UI.Command.ComCommands
```

Comment:
Nitpick - Is there any reason that this needs to be in its own namespace? The folder obviously makes sense organizationally, but sub-dividing the namespace doesn't seem like it does much other than adding more noise to the usings - I'm having difficulty seeing this as a qualitatively different enough implementation to warrant that.

Reply:
The primary reasoning was to make it easy to write a code analysis rule that permits COM access only in certain namespaces. That doesn't exist yet but I have had in mind to write one in future.